lib/ansiblelint/utils.py

```diff
@@ -712,9 +716,15 @@ def get_playbooks_and_roles(options=None):
 
     for p in map(Path, files):
 
-        if any(str(p).startswith(file_path) for file_path in options.exclude_paths):
+        try:
+            for file_path in options.exclude_paths:
+                if str(p.resolve()).startswith(str(file_path)):
```

Comment:
Since you dropped py3.5 this could probably be simplified.

Reply:
I look and it seems that not.